lib/dao.js

```diff
@@ -367,27 +394,52 @@ export class Organization {
   }
 
   /**
-   * Returns the name of the Organization as stored in the Avatar
+   * Returns the name of the DAO as stored in the Avatar
    * @return {string}
    */
   async getName() {
-    return getWeb3().toAscii(await this.avatar.orgName());
+    return Utils.getWeb3().toUtf8(await this.avatar.orgName());
   }
 
   /**
-   * Returns the token name for the Organization as stored in the native token
+   * Returns the token name for the DAO as stored in the native token
    * @return {string}
    */
   async getTokenName() {
     return await this.token.name();
   }
 
   /**
-   * Returns the token symbol for the Organization as stored in the native token
+   * Returns the token symbol for the DAO as stored in the native token
    * @return {string}
    */
   async getTokenSymbol() {
     return await this.token.symbol();
   }
 
+  /**
+   * Returns promise of the DAOstack (Genesis) avatar address, or undefined if not found
```

Comment:
why just return the address and not a Dao object?

Reply:
> why just return the address and not a Dao object?

To avoid the overhead involved, allowing the user do choose what they want to do with it.  Easy enough for them to do `DAO.at(genesisAddress)`.

Also a bit of SOC.